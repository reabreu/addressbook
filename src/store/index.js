import { configureStore } from "@reduxjs/toolkit";
import usersReducer from "../features/users/users-slice";
import settingsReducer from "../features/settings/settings-slice";

export default configureStore({
  reducer: {
    settings: settingsReducer,
    users: usersReducer,
  },
});
