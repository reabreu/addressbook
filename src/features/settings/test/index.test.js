import React from "react";
import { render } from "@testing-library/react";
import { Provider } from "react-redux";
import Settings from "../";
import store from "../../../store";

describe("<Settings /> spec", () => {
  it("Matches snapshot", () => {
    const { asFragment } = render(
      <Provider store={store}>
        <Settings />
      </Provider>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
