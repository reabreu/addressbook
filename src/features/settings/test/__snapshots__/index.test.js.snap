// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Settings /> spec Matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm jnqzAZ"
  >
    <p>
      Your active language is GB, use the select box bellow to change this
    </p>
    <select>
      <option
        value="CH"
      >
        CH
      </option>
      <option
        value="ES"
      >
        ES
      </option>
      <option
        value="FR"
      >
        FR
      </option>
      <option
        selected=""
        value="GB"
      >
        GB
      </option>
    </select>
  </div>
</DocumentFragment>
`;
